app/models/concerns/orchestration/tftp.rb

```diff
@@ -100,6 +119,7 @@ def setTFTPBootFiles
     host.operatingsystem.pxe_files(host.medium, host.architecture, host).each do |bootfile_info|
       for prefix, path in bootfile_info do
         valid = false unless tftp.fetch_boot_file(:prefix => prefix.to_s, :path => path)
+        valid = false unless tftp6.fetch_boot_file(:prefix => prefix.to_s, :path => path)
```

Comment:
Can `tftp`/`tftp6` be de-duplicated to avoid two requests if both v4/v6 subnets reference the same smart proxy? The TFTP file fetching is unreliable when called twice as it downloads simultaneously. It may also make for clearer log files on deletion (etc) where you'd see one successful request, followed by an unsuccessful request.

`[tftp, tftp6].uniq { |p| p.url }.each { |p| p.fetch_boot_file .... }` might do it crudely?